crates/subspace-gateway/src/commands/run.rs

```diff
@@ -1,6 +1,9 @@
 //! Gateway run command.
 //! This is the primary command for the gateway.
 
+mod dsn;
```

Comment:
I would call this just `network`. It is already like that in farmer, but not in node yet.